Habinezajanvier/Capstone
# #37 edit article

#### What does this PR do?
Enable article editing
#### Description of Task to be completed?
- adding form for editing article
- getting articles's value from HTML
- Submitting edited article to firebase DB
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the directory `cd my-brand-project`
- Checkout the branch `git checkout ft-blog-connection-#34`
- Using your preferred browser start `index.html` found in `/admin/` folder
- Click on article link found in left-side nav-bar
- on the displayed article list, click on edit button of any article
- On the provided form edit your article as you want
> After submitting your changes you should see your article edited
#### Any background context you want to provide?
If `/admin/index.html` redirect you to login use **`admin@email.com`** and **admin123** as credential
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/ozBwQ2Db/37-enable-edit-item) for relevant pivotal tracker stories
